### PR TITLE
Add the note forked from gogs

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 ## Purpose
 
-The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across **all platforms** that Go supports, including Linux, macOS, and Windows on x86, amd64, ARM and PowerPC architectures. Want to try it before doing anything else? Do it [online](https://try.gitea.io/)!
+The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across **all platforms** that Go supports, including Linux, macOS, and Windows on x86, amd64, ARM and PowerPC architectures. Want to try it before doing anything else? Do it [online](https://try.gitea.io/)! This project is forked from [Gogs](https://gogs.io) for the [reason](https://blog.gitea.io/2016/12/welcome-to-gitea/).
```
Comment (tboerger) [inline README.md]: Just put the reasoning link on the fork word

Comment (lunny) [reply README.md]: Done.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by strk** [comment]
Sorry but that long line makes it very hard for me to check
what changed from the emailed diff :(

I'll LGTM on trust, but please please please consider splitting
long lines to make diffs smaller and easier to review.

**Pull request merged**